UefiCpuPkg/PiSmmCpuDxeSmm/CpuS3.c

```diff
@@ -1134,6 +1134,12 @@ GetAcpiCpuData (
   // For a native platform, copy the CPU S3 data into SMRAM for use on CPU S3 Resume.
   //
   CopyMem (&mAcpiCpuData, AcpiCpuData, sizeof (mAcpiCpuData));
+  // MU_CHANGE [BEGIN] - CodeQL change
+  if (&mAcpiCpuData == 0) {
+    ASSERT (&mAcpiCpuData == 0);

```

Comment:
Was this assert meant to be `!= 0`?

```suggestion
    ASSERT (&mAcpiCpuData != 0);
```

Reply:
This was removed.